e2e/pages/adf/demo-shell/process-services/taskDetailsCloudDemoPage.ts

```diff
@@ -35,6 +38,35 @@ export class TaskDetailsCloudDemoPage {
         return this;
     }
 
+    formFields() {
+        return new FormFields();
+    }
+
+    checkSaveButtonIsDisplayed() {
+        BrowserVisibility.waitUntilElementIsVisible(this.saveButton);
+        return this;
+    }
+
+    checkCompleteButtonIsDisplayed() {
+        BrowserVisibility.waitUntilElementIsVisible(this.completeButton);
+        return this;
+    }
+
+    checkCompleteButtonIsNotDisplayed() {
+        BrowserVisibility.waitUntilElementIsVisible(this.completeButton);
+        return this;
+    }
+
+    clickSaveButton() {
+        BrowserActions.click(this.saveButton);
+        return this;
+    }
+
+    clickCompleteButton() {
+        BrowserActions.click(this.completeButton);
+        return this;
+    }
+
```

Comment:
Methods related to complete button are already implemented in task-form-cloud-component.page.ts. Can we add the save button ones there ? We can also add a method here getTaskForm() if needed, that will create a new instance of task-form-cloud-component.

Reply:
the locator for the buttons on task-form-cloud-component.page is different from the one on the demo page. Hence added them on demo page.

Reply:
Yes I realised that yesterday. Complete button has a different locator when it's a task with a form and a task without. But I think it's better to add the id in task-form-cloud.component.html and therefore being able to use the same locator, rather than having similar methods in different pages.